opentelemetry-sdk/tests/metrics/test_measurement_consumer.py

```diff
@@ -180,7 +180,11 @@ def sleep_1(*args, **kwargs):
             -1
         ].kwargs["timeout_millis"]
 
+        self.assertAlmostEqual(
+            round(10000 - callback_options_time_call), 1000.0, places=1
```

Comment:
@mdcg, what do you think about using delta here instead of places?

Reply:
Sounds good.